default.nix

```diff
@@ -20,6 +20,9 @@ mkDerivation {
     hnix io-streams optparse-applicative temporary text transformers
     unordered-containers
   ];
+  shellHook = ''
```

Comment:
Please splice this into the derivation (`overrideCabal`) from `shell.nix` instead so `default.nix` is always just the `cabal2nix` output.